draft-ietf-quic-http.md

```diff
@@ -671,14 +673,16 @@ treat the presence of the same parameter more than once as a connection error of
 type HTTP_MALFORMED_FRAME.
 
 The payload of a SETTINGS frame consists of zero or more parameters, each
-consisting of an unsigned 16-bit setting identifier and a value which uses the
+consisting of a setting identifier and a value, each of which uses the
```

Comment:
I'd put it as "...identifier and a value, each encoded as a QUIC variable-length integer," simply because I don't see how the identifier and the value _use_ the encoding.

(Again, this is not a new thing in this change, but simply something I've noticed.)

Reply:
"both encoded as" would work too